styles/blocks/stores/stores.less

```
@@ -0,0 +1,90 @@
+.stores {
+    padding-left: 0;
+    list-style-type: none;          /* Для отключения списка */
+    display: block;
+    margin: 0 -5px 0 -5px;
+    height: 40px;
+
+    &__item{
```

не забывайпро пробел после селектора. по всему реквесту посмотри 

```suggestion
    &__item {
```

---

Принято. Но пока хороший и рабочий линтер не подберу - так и будет :(

---

не стоит постоянно полагаться на линтер - ты должен научиться видеть такие вещи на автомате. Тем более, если у тебя нет линтера, нужно внимательно следить за своим кодстайлом